SkietbaanBE/SkietbaanBE/appsettings.json

```diff
@@ -13,6 +13,6 @@
     }
   },
   "ConnectionStrings": {
-    "SkietbaanDatabase": "Server=dev.retrotest.co.za;Database=skietbaan;User Id=group3;Password=3c~$+DQk3u)5S*S<;"
+    "SkietbaanDatabase": "Server=dev.retrotest.co.za;Database=skietbaan;User Id=group3;Password=Y%6KG-bmr96Pe5rC"
```

Comment:
Didnt henko say not to commit db connection details?

Reply:
Aha its the wrong connection i found a way to hide the correct details